src/activemasternode.cpp

```diff
@@ -14,6 +14,62 @@
 #include "netbase.h"
 #include "protocol.h"
 
+OperationResult errorOut(const std::string& errorStr)
```

Comment:
#1798 also defines an `errorOut()` function in `sapling_operation.cpp` that conflicts with this.